EIPS/eip-draft_increase_calldata_cost.md

```diff
@@ -0,0 +1,43 @@
+---
+eip: <tbd>
```

Comment:
```suggestion
eip: 7703
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename